IFC4x3/Properties/r/RQD_2o32BT3ZX4bO$w85z$rZ25/Documentation.md

```diff
@@ -0,0 +1 @@
+Quotient of the cumulative length of cores with a length greater than 10 cm, by the total length of the core pass with a length greater than or equal to 1m.
```

Comment:
Not sure what is meant by "cores".

Reply:
Maybe an elaboration of the description is needed?

Reply:
Renamed + changed "cores" to "drill cores".